microsoft-edge/devtools-guide-chromium/evaluate-performance/share-traces.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Share performance and memory traces
+description: Learn to export and import performance and memory traces in DevTools to get help when investigating issues.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 01/06/2023
+---
+# Share performance and memory traces
+
+The **Performance** and **Memory** tools record runtime data about your web page. Exploring the recorded data makes it possible to improve your web page's memory usage or runtime performance.
```

Comment:
Global: change "web page" to "webpage".
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/page

FTS style guide: 
"web page" 1 hit.
"webpage" 40 hits.

Not properly doc'd in style guide but hit counts show "web page" is style guide's standard.